### PR TITLE
[chore] Change the organizers section to "Founders"

### LINKED ISSUE #7: [chore] Change the organizers section to "Founders"
The organizers section is under [about page](http://oaxacarb.org/about.html)

Image

### PR DESCRIPTION
**Pull request by JuanVqz**
Fixes https://github.com/oaxacarb/oaxacarb.github.io/issues/7

### CODE REVIEW

File: source/about.html.haml [hunk 1]
```diff
@@ -8,22 +8,22 @@ title: Acerca de
...
   %ul
     %li
-      = link_to 'apux', 'https://github.com/apux'
+      = link_to 'apux', 'https://github.com/apux', target: :_blank
     %li
-      = link_to 'fervisa', 'https://github.com/fervisa'
+      = link_to 'fervisa', 'https://github.com/fervisa', target: :_blank
     %li
-      = link_to 'garcialopezco', 'https://github.com/fervisa'
+      = link_to 'garcialopezco', 'https://github.com/garcialopezco', target: :_blank
```
Comment (JuanVqz) [inline source/about.html.haml:24]: Also, I fixed this link, it was linking to the wrong GitHub account

File: source/about.html.haml [hunk 2]
```diff
@@ -8,22 +8,22 @@ title: Acerca de
...
     %li
-      = link_to 'thotmx', 'https://github.com/thotmx'
+      = link_to 'thotmx', 'https://github.com/thotmx', target: :_blank
```
Comment (JuanVqz) [inline source/about.html.haml:26]: And added the target blank to keep the viewer on the page

**Comment by thotmx** [comment]
Looks good!

**Review by thotmx — APPROVED** [review]
Looks good to me. We should wait to deploy this waiting a response from the founders. I already asked in Slack

**Comment by JuanVqz** [comment]
> Looks good to me. We should wait to deploy this waiting a response from the founders. I already asked in Slack

Let's wait but seems to be approved.
Fernando added a 👍 emoji.

**Comment by thotmx** [comment]
I asked Carlos, and he said it's Ok. We just need ApuX confirmation and we are done.

**Pull request merged**